Riot/Modules/Analytics/DecryptionFailure.swift

```diff
@@ -16,12 +16,10 @@
 
 import AnalyticsEvents
 
-/// Failure reasons as defined in https://docs.google.com/document/d/1es7cTCeJEXXfRCTRgZerAM2Wg5ZerHjvlpfTW-gsOfI.
```

Comment:
Looking at this doc it seems quite outdated, I do not see the value of continuing to reference it.